contrib/epee/src/mlog.cpp

```diff
@@ -338,11 +338,21 @@ bool is_stdout_a_tty()
   return is_a_tty.load(std::memory_order_relaxed);
 }
 
+static bool is_nocolor()
+{
+  static const char *no_color_var = getenv("NO_COLOR");
+  static const bool no_color = no_color_var && *no_color_var; // apparently, NO_COLOR=0 means no color too (as per no-color.org)
```

Comment:
You should use `strtol` to check the actual numeric value if you want to compare with 0.

Reply:
I think it's better to go with the example implementation:
```
if (no_color != NULL && no_color[0] != '\0') {
  color = false;
}
```

Reply:
I don't want to compare with 0, 0 apparently means the same as 1 or yes or no (according to the page). The contents do not matter, only their existence.

Reply:
Yes, I've just read the docs too and it explicitly say "any non-empty string", so it's better to leave the code as is.